specification/eventgrid/data-plane/Microsoft.Resources/stable/2018-01-01/Resources.json

```diff
@@ -41,19 +41,20 @@
         },
         "authorization": {
           "description": "The requested authorization for the operation.",
-          "type": "string"
+          "$ref": "#/definitions/ResourceAuthorization"
         },
         "claims": {
           "description": "The properties of the claims.",
-          "type": "string"
+          "type": "object",
+          "additionalProperties": true
```

Comment:
"additionalProperties": {
            "type": "string"
          }

Reply:
I think these should probably be "object" in case the values are objects.